azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -196,6 +196,7 @@ public void run() {
         logger.warn("Shutting down executor...");
         try {
           app.shutdownNow();
+          app.getFlowRunnerManager().deleteExecutionDirectory();
```

Comment:
Is the shutdown hook invoked in the orderly shutdown process? How did you test it?

The deletion could be made internal to the flowRunnerManager in the previous commits. Why does it need to be made public?